### ISSUE TITLE
DM screen blank on Linux

### ISSUE THREAD

**Issue by maxb2** [apclary/dungeon-revealer#183]
Since [v1.2.0](https://github.com/apclary/dungeon-revealer/releases/tag/v1.2.0) for linux the DM page is blank. When navigating directly to localhost:3000/dm, the browser asks for a password normally and then loads a blank page. Navigating to localhost:3000 shows the landing page correctly. However, clicking the DM link exhibits the same behavior as before.
I tested this on Ubuntu 18.10 with Firefox and Chromium using both normal and incognito modes. I also tried both the executable and running from source.

The last version that works correctly is [v1.1.0](https://github.com/apclary/dungeon-revealer/releases/tag/v1.1.0)

dungeon-revealer log:
```
Starting dungeon-revealer.

DM Section: http://localhost:3000/dm
Player Section: http://128.206.125.12:3000

GET /dm 401 0.982 ms - 0
express deprecated res.sendfile: Use res.sendFile instead ../../../../../../../snapshot/dungeon-revealer/app.js:411:7
GET /dm 200 4.877 ms - 3024
GET /static/css/main.3deeba78.chunk.css 200 0.648 ms - 1574
GET /static/js/main.666295cc.chunk.js 200 0.785 ms - 989
GET /static/js/5.0e6446a8.chunk.js 200 0.847 ms - 132428
GET /static/js/0.25c66a6b.chunk.js 200 1.220 ms - 153774
GET /static/js/1.b234b132.chunk.js 200 1.528 ms - 20431
GET /static/js/6.c6b9d1ab.chunk.js 200 1.186 ms - 60600
GET /static/js/4.64e68464.chunk.js 200 1.465 ms - 167376

```

Page source of the blank DM page:
```
<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8" />
    <link rel="shortcut icon" href="/favicon.ico" />
    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no" />
    <meta name="theme-color" content="#000000" />
    <link rel="manifest" href="/manifest.json" />
    <title>Dungeon Revealer</title>
    <link href="/static/css/main.3deeba78.chunk.css" rel="stylesheet">
</head>

<body id="body"><noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>
    <div id="modal"></div>
    <script>
        !function(c){function e(e){for(var r,t,n=e[0],o=e[1],u=e[2],a=0,i=[];a<n.length;a++)t=n[a],Object.prototype.hasOwnProperty.call(f,t)&&f[t]&&i.push(f[t][0]),f[t]=0;for(r in o)Object.prototype.hasOwnProperty.call(o,r)&&(c[r]=o[r]);for(d&&d(e);i.length;)i.shift()();return p.push.apply(p,u||[]),l()}function l(){for(var e,r=0;r<p.length;r++){for(var t=p[r],n=!0,o=1;o<t.length;o++){var u=t[o];0!==f[u]&&(n=!1)}n&&(p.splice(r--,1),e=s(s.s=t[0]))}return e}var t={},f={3:0},p=[];function s(e){if(t[e])return t[e].exports;var r=t[e]={i:e,l:!1,exports:{}};return c[e].call(r.exports,r,r.exports,s),r.l=!0,r.exports}s.e=function(o){var e=[],t=f[o];if(0!==t)if(t)e.push(t[2]);else{var r=new Promise(function(e,r){t=f[o]=[e,r]});e.push(t[2]=r);var n,u=document.createElement("script");u.charset="utf-8",u.timeout=120,s.nc&&u.setAttribute("nonce",s.nc),u.src=s.p+"static/js/"+({}[o]||o)+"."+{0:"25c66a6b",1:"b234b132",4:"64e68464",6:"c6b9d1ab",7:"8069ccca"}[o]+".chunk.js";var a=new Error;n=function(e){u.onerror=u.onload=null,clearTimeout(i);var r=f[o];if(0!==r){if(r){var t=e&&("load"===e.type?"missing":e.type),n=e&&e.target&&e.target.src;a.message="Loading chunk "+o+" failed.\n("+t+": "+n+")",a.name="ChunkLoadError",a.type=t,a.request=n,r[1](a)}f[o]=void 0}};var i=setTimeout(function(){n({type:"timeout",target:u})},12e4);u.onerror=u.onload=n,document.head.appendChild(u)}return Promise.all(e)},s.m=c,s.c=t,s.d=function(e,r,t){s.o(e,r)||Object.defineProperty(e,r,{enumerable:!0,get:t})},s.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},s.t=function(r,e){if(1&e&&(r=s(r)),8&e)return r;if(4&e&&"object"==typeof r&&r&&r.__esModule)return r;var t=Object.create(null);if(s.r(t),Object.defineProperty(t,"default",{enumerable:!0,value:r}),2&e&&"string"!=typeof r)for(var n in r)s.d(t,n,function(e){return r[e]}.bind(null,n));return t},s.n=function(e){var r=e&&e.__esModule?function(){return e.default}:function(){return e};return s.d(r,"a",r),r},s.o=function(e,r){return Object.prototype.hasOwnProperty.call(e,r)},s.p="/",s.oe=function(e){throw console.error(e),e};var r=window["webpackJsonpdungeon-revealer"]=window["webpackJsonpdungeon-revealer"]||[],n=r.push.bind(r);r.push=e,r=r.slice();for(var o=0;o<r.length;o++)e(r[o]);var d=n;l()}([])
    </script>
    <script src="/static/js/5.0e6446a8.chunk.js"></script>
    <script src="/static/js/main.666295cc.chunk.js"></script>
</body>

</html>
```

**Comment by n1ru4l** [comment]
Can, confirm this! had to delete my data folder (no map must be available).

Working on a fix right now.

**Comment by n1ru4l** [comment]
will be fixed in #185
